server/models/db/appointment.js

```diff
@@ -47,8 +47,27 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     return db.any(`update ${Tables.appointment} set status='${AppointmentStatus.cancelled}' from ${Tables.appointment} a1 inner join ${Tables.workerTimeslot} w1 on a1.worker_timeslot_id = w1.worker_timeslot_id ${queryCondition};`);
 }
 
+// Cancels specific appointments/meeting and updates worker availability to available for a worker for the specific timeslot. 
+// Note that appointments/meetings are synonymous, but only appointments will be used in the backend to maintain consistency.
+async function cancelSpecificAppointment(appointmentId) {
+    //how to know if update was successful? 
```

Comment:
Hi Peter,
I would suggest adding more to this comment or removing it to avoid future confusion.

Reply:
Resolved!